pages/fundamentals/data-types.mdx

```diff
@@ -772,6 +773,69 @@ and display it nicely in the client-side results.
 
 </Callout>
 
+## Point
+
+Point is a spatial data type consisting of 2D or 3D locations in the Cartesian
+or WGS84 system. The coordinates of the location are stored as a 64-bit `Float`
+numbers. Each point type has an associated coordinate reference system (CRS)
+and a spatial reference identifier (SRID). Points can be created with the
+[point function](/querying/functions#spatial). For fast queries points can
+leverage the [spatial index](/fundamentals/indexes#spatial-index).
+
+### Coordinate Reference Systems (CRS)
+
+Memgraph supports four Coordinate Reference Systems (CRS) for spatial data, divided into two categories:
+
+1. **Geographic CRS** models points on the Earth’s surface using longitude and latitude based on the [WGS84](https://epsg.io/4326) system.
+2. **Cartesian CRS** models points in a Euclidean space using standard Cartesian coordinates.
```

Comment:
We have Geographic and Cartesian CRS here. The title below should follow this and be Geographic instead of WGS84. Link to the sections below and make them h4.